src/server/api/campaign.js

```diff
@@ -111,6 +111,9 @@ export const resolvers = {
       r.table('assignment')
         .getAll(campaign.id, { index: 'campaign_id' })
         .eqJoin('user_id', r.table('user'))('right')
+        .eqJoin('user_id', r.table('user_organization')).zip()
+        .filter({ organization_id: campaign.organization_id })
```

Comment:
Do these two lines seem ok? I wanted to be able to access the `role` from `campaign.texters`

Reply:
Can we convert this to a knex query?  I think it'll be more straightforward there.
Also, not sure it's relevant, but you might want to look at this:
https://github.com/MoveOnOrg/Spoke/blob/scaling-messages/src/server/api/user.js#L4
seems like if we are doing user queries by org with similar aims, maybe worth using here, too.

Reply:
Also, the join data won't actually be used, unless you leverage it in this method:
https://github.com/MoveOnOrg/Spoke/blob/scaling-messages/src/server/api/user.js#L39
i.e. something like `user.orgRoles ? getRoles(user.orgRoles[organizationId]) :  ...`